pkg/mapper/configmap/configmap.go

```diff
@@ -0,0 +1,210 @@
+package configmap
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/sirupsen/logrus"
+	"gopkg.in/yaml.v2"
+	core_v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/fields"
+	utilyaml "k8s.io/apimachinery/pkg/util/yaml"
+	"k8s.io/apimachinery/pkg/watch"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/kubernetes/typed/core/v1"
+	"k8s.io/client-go/tools/clientcmd"
+	"sigs.k8s.io/aws-iam-authenticator/pkg/config"
+)
+
+type MapStore struct {
+	mutex sync.RWMutex
+	users map[string]config.UserMapping
+	roles map[string]config.RoleMapping
+	// TODO: Use kubernetes set.
```

Comment:
Is this something we want to do now?

Reply:
not much advantage to doing so as we don't do any set operations (TODO was left by someone else). Will leave TODO intact though.

Reply:
If we agree that there is no advantage, let's remove the TODO to reduce confusion.